tests/search_disk_index.cpp

```diff
@@ -336,6 +363,15 @@ int main(int argc, char** argv) {
                        po::bool_switch()->default_value(false),
                        "Include full precision data in the index. Use only in "
                        "conjuction with compressed data on SSD.");
+    desc.add_options()(
+        "num_extra_start_points",
+        po::value<uint32_t>(&num_extra_start_points)->default_value(0),
+        "Number of extra starting points used for searching index (defaults to 0");
```

Comment:
Could you please deafult it to the num_Extra_Start_points used in the build index and stored to file?